content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,158 @@
+  `dvc exp init` may fail if the parameters file does not exist at the time of
+  the invocation, as DVC reads the file to find parameters to track for the
+  stage.
+
+- `--metrics` - override the path to metrics file to track, which your
+  experiment produces. Default is `metrics.json` file.
+
+- `--plots` - override the path to plots file or directory, which your
+  experiment produces. The default is `plots`.
+
+- `--live` - override the directory `path` for [DVCLive](/doc/dvclive), which
+  your experiment will write logs to. The default is `dvclive` directory, which
+  only comes to effect when used with `--type=live`.
+
+- `--type` - selects the type of the stage to create. Currently it provides two
+  different kinds of stages: `default` and `live`. If unspecified, `default`
+  stage is created.
+
+  `default` stage creates a stage with `metrics` and `plots` tracked by DVC
+  itself, and does not track live-created artifacts (unless explicitly
+  specified).
+
+  `live` stage is intended for use in deep-learning scenarios, where metrics and
+  plots are tracked by [dvclive](/doc/dvclive) and supports tracking progress
+  while training a deep-learning model with
+  [checkpoints](/doc/command-reference/exp/run#checkpoints).
+
+- `-n <stage>`, `--name <stage>` - specify a custom name for the stage generated
+  by this command (e.g. `-n train`). By default, the name of the stage depends
+  on `--type` of the stage that is being created. If
+  `--type=default, the name of the stage will be `default`, and in case of `--type=live`, the name of the stage will be `live`.
+
+  Note that the stage name can only contain letters, numbers, dash `-` and
+  underscore `_`.
+
+- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
+  asking for confirmation.
+
+- `--run` - runs the experiment after initializing it.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+### Setting up custom paths
+
+`dvc exp init` supports
+[setting up custom workspace paths by setting up DVC config](/doc/command-reference/config#exp),
+where you can add an `exp` section to the config file to point to the paths for
+your code, data, models, parameters, metrics, plots (either images or tabular
+data to be plotted), and dvclive outputs.
+
+```dvc
+$ dvc config exp.data datasets/
+$ dvc config exp.params config.yaml
+$ dvc config exp.code scripts/train.py
+$ dvc config exp.models trained_models/
+$ dvc config exp.metrics reports.json
+$ dvc config exp.plots viz/
+```
+
+You can leave some configurations to use default values. This can be useful in a
+system or global config so that you can avoid repeating these paths if all of
+your projects share a similar structure.
+
+## Non-interactive mode
+
+## Guided/Interactive mode
+
```

Comment:
I don't really want to show all the prompts. And I think most of the information are already mentioned elsewhere, so I am thinking of leaving these fields.

Reply:
Good call. If we ever make a detailed guide we can include all prompts but it's probably unnecessary either way.